src/components/Tokens/TokensBanner.tsx

```diff
@@ -59,23 +60,30 @@ const Description = styled(Link)`
 export default function TokensBanner() {
   const theme = useTheme()
   const [showTokensPromoBanner, setShowTokensPromoBanner] = useShowTokensPromoBanner()
+  const navigate = useNavigate()
+  const chainName = useGlobalChainName().toLowerCase()
 
-  const closeBanner = () => {
-    setShowTokensPromoBanner(false)
+  const openExplorePage = () => {
```

Comment:
nit: `open` kinda makes this sound like a popup

Reply:
true, changed to navigate